webui/src/pages/setup/communicationPreferences.tsx

```diff
@@ -18,18 +18,23 @@ export const CommunicationPreferencesSetup: FC<CommunicationPreferencesSetupProp
     disabled,
 }) => {
     const [userEmail, setUserEmail] = useState<string>("");
+    const [adminUser, setAdminUser] = useState<string>("admin");
     const [updatesCheck, setUpdatesCheck] = useState<boolean>(false);
-    const [securityCheck, setSecurityCheck] = useState<boolean>(false);
+    const [securityCheck, setSecurityCheck] = useState<boolean>(true);
```

Comment:
This was unchecked for a reason. Having it checked by default may have legal ramifications. IANAL (obviously and happily), but I'd suggest consulting with @ozkatz.

Reply:
@ozkatz can you comment here? Since we're saying it's recommended for users to have security updates then pre-selecting it for them might be useful.

Reply:
I don't know of any legal ramifications, but we did want to keep communications from the lakeFS team to be opt-in at this points. 

@rmoff I think we can keep the defaults the same and merge this (very useful!!) PR first - and perhaps open a separate discussion if we want to revisit the opt-in vs opt-out decision.